Actions/PullPowerPlatformChanges/GitCloneReponsitory.ps1

```diff
@@ -0,0 +1,25 @@
+Param(
+    [Parameter(HelpMessage = "The GitHub actor running the action", Mandatory = $false)]
+    [string] $actor,
+    [Parameter(HelpMessage = "The GitHub token running the action", Mandatory = $false)]
+    [string] $token,
+    [Parameter(HelpMessage = "Set the branch to update", Mandatory = $false)]
+    [string] $updateBranch,
+    [Parameter(HelpMessage = "Direct Commit", Mandatory = $false)]
+    [bool] $directCommit
+)
+
+$ErrorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
+
+Write-Host "Starting GitCloneRepository.ps1 with parameters: $([environment]::Newline)actor: $actor$([environment]::Newline)token: $token$([environment]::Newline)updateBranch: $updateBranch$([environment]::Newline)directCommit: $directCommit"
```

Comment:
`$token` should probably be removed from here.

Reply:
agree